data/hai/hai missions.txt

```diff
@@ -93,6 +104,14 @@ event "label hai space"
 	galaxy "label hai"
 		sprite "label/hai"
 
+mission "Discovered Ultima Thule"
+	landing
+	invisible
+	to complete
+		never
+	on enter "Ultima Thule"
+		fail
```

Comment:
Why not set a custom condition in the `on enter`? Then you don't have to deal with what looks like a negative case and treat that as a positive.